CSH/test_plots/test/test_plot_msid_latest_conda.py

```diff
@@ -0,0 +1,34 @@
+#!/proj/sot/ska3/flight/bin/python
+
+import sys,os
+
+#Path altering to import script which is undergoing testing
+TEST_DIR = os.path.dirname(os.path.realpath(__file__))
+PARENT_DIR = os.path.dirname(TEST_DIR)
+OUT_DIR = f"{TEST_DIR}/outTest"
+sys.path.insert(0,PARENT_DIR)
+LIVE_BIN_DIR = "/data/mta4/www/CSH/test_plots"
+
+NEW_DIR_LIST = [OUT_DIR]
+for dir in NEW_DIR_LIST:
+	os.system(f"mkdir -p {dir}")
+
+#Extra Parameter Setup
+#TEST_MSID_GROUP_SELECTION = ['Sys_temps', 'Spcelec']
+
+import plot_msid_latest_conda as pml
+import soh_msid_plot_class_v3 as smpc
+import plot_cleaning as pc
+
+MOD_GROUP = [pml, smpc, pc]
+#Permute across imported modules to change pathing variables
+for mod in MOD_GROUP:
+    if hasattr(mod,'BIN_DIR'):
```

Comment:
Consider using argparse so that a test case run can be defined by a command line argument, and BIN_DIR, OUT_DIR can be assigned accordingly. This would eliminate the need for hardcoding "test_plots" directory in the names of BIN_DIR, OUT_DIR paths, and the need to change these definitions in the code at the time of installation. 

For example, if test.py contains the following lines, then a call "python test.py -t test" will redefine OUT_DIR accordingly. All other calls, including no arguments, will use the default path.

```
import argparse
OUT_DIR = "/data/mta4/www/CSH/plots/plot_sections"

if __name__ == "__main__":
    parser = argparse.ArgumentParser()
    parser.add_argument("-t", "--test", required=False)
    args = parser.parse_args()
    is_test = args.test
    if is_test == "test":
        OUT_DIR = "/data/mta4/www/CSH/test_plots/plot_sections"
    print(is_test, OUT_DIR)
```

Finally, it may be desired to use a more robust setup like:

```
parser.add_argument("-m", "--mode", required=True)
args = parser.parse_args()
if args.mode not in ['flight', 'test']:
    raise Exception("Mode should be 'flight' or 'test'")
if args.mode == 'flight':
    OUT_DIR = 
elif args.mode == 'test':
    OUT_DIR = 
```

Reply:
Testing via command line now included

Reply:
Quick clarification as mentioned in a separate Repos PR. the reason that this script will only have arg.mode conditional statements for 'test' and 'flight' mode is because the arg parser library will automatically throw an error at the user if one of these options is not selected, therefore there's no need to include extra conditionals to deal with a different running mode being passed.